tensorboard/webapp/widgets/line_chart_v2/sub_view/line_chart_axis_view.ts

```diff
@@ -29,6 +29,16 @@ import {
 
 const DAY_IN_MS = 24 * 1000 * 60 * 60;
 
+interface MinorTick {
+  value: number;
+  tickFormattedString: string;
+}
+
+interface MajorTick {
+  start: number;
```

Comment:
Could you explain more about 'start' vs 'value'? why are they named differently?